src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationsImplementation.cs

```diff
@@ -11,6 +13,8 @@
 using Windows.Foundation;
 using Windows.Storage.Streams;
 
+using SREMemoryStream = System_Runtime_Extensions::System.IO.MemoryStream;
```

Comment:
I will. For the record here you can't use the same name for the using as a type that is in scope.  The code in this file is in the System.IO namespace, so defining as MemoryStream (which also is in System.IO) isn't possible. (It will give some very strange compile errors.)